openmetadata-ui/src/main/resources/ui/cypress/common/advancedSearch.js

```diff
@@ -334,20 +350,36 @@ export const addTag = (tag) => {
     .scrollIntoView()
     .click();
 
-  cy.get('[data-testid="tag-selector"]').should('be.visible').click().type(tag);
+  let tagsArray = !isArray(tag) ? [tag] : tag;
 
-  cy.get('.ant-select-item-option-content')
-    .contains(tag)
-    .should('be.visible')
-    .click();
+  tagsArray.map((tagName) => {
+    cy.get('[data-testid="tag-selector"]')
+      .should('be.visible')
+      .click()
+      .type(tagName);
 
-  cy.get('[data-testid="tag-selector"] > .ant-select-selector').contains(tag);
+    const tagDisplayName = tagName.split('.').reverse()[0];
+
+    cy.get('.ant-select-item-option-content')
+      .contains(tagDisplayName)
+      .should('be.visible')
+      .click();
+
+    cy.get('[data-testid="tag-selector"] > .ant-select-selector').contains(
+      tagName
+    );
+  });
 
   cy.get('[data-testid="saveAssociatedTag"]').should('be.visible').click();
-  cy.get('[data-testid="entity-tags"]')
-    .scrollIntoView()
-    .should('be.visible')
-    .contains(tag);
+
+  tagsArray.map((tagName) => {
+    cy.get('[data-testid="entity-tags"]')
+      .scrollIntoView()
+      .should('be.visible')
+      .contains(tagName);
+  });
+
+  cy.log('exit');
```

Comment:
log?

Reply:
Sorry, I missed that. I'll take care of it